utilities/test_suite/HIP/runTests.py

```diff
@@ -35,6 +36,53 @@
 inFilePath2 = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src2')
 qaInputFile = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src1')
 
+import os
+import subprocess  # nosec
+import argparse
+import sys
+import datetime
+import shutil
```

Comment:
Why is the same set imports being imported twice?

Reply:
Done